client/src/app/containers/Alert/types.ts

```diff
@@ -0,0 +1,3 @@
+export type AlertType = 'error' | 'warning' | 'success' | 'info';
+
+export const alertTypeToStyle = (t: AlertType) => `alert-${t}`;
```

Comment:
:student: What are these doing?

Reply:
`alertTypeToStyle` is a function which takes an `AlertType` and converts it to a string.

the `alert-${t}` is javascript/typescript's syntax for string interpolation.